### ISSUE TITLE
Recognize from a local file

### ISSUE THREAD

**Issue by linsui** [woheller69/whoBIRD#53]
Currently it can only use real-time record. Can you add a feature so that it can use a local file? Thanks!

**Comment by woheller69** [comment]
you can use a music player which runs in background and plays via your speakers and switch on whoBIRD.

**Comment by linsui** [comment]
Yes, but it would be better to load the file directly...

**Comment by linsui** [comment]
Yes, right, but I thought most users only need what Android supports anyway. :)

**Comment by blaiseli** [comment]
I too would find it useful to be able to analyze a local file. Example I have in mind: feeding whoBird .wav files saved by Merlin and experiment with frequency filter to compare results.

**Comment by nishantnnb** [comment]
+1
